packages/main/src/MenuItem.ts

```diff
@@ -202,6 +205,33 @@ class MenuItem extends ListItem implements IMenuItem {
 	@i18n("@ui5/webcomponents")
 	static i18nBundle: I18nBundle;
 
+	_itemNavigation: ItemNavigation;
+
+	constructor() {
+		super();
+
+		this._itemNavigation = new ItemNavigation(this, {
+			navigationMode: NavigationMode.Horizontal,
+			behavior: ItemNavigationBehavior.Static,
+			getItemsCallback: () => this._navigableItems,
+		});
+	}
+
+	get _navigableItems(): Array<HTMLElement> {
+		return [...this.endContent];
```

Comment:
What would happen if the user passes non-interactive items in `endContent`? I saw the documentation says we expect button, link or icon( I presume button-like icon, i.e. clickable). But will there be an issue if they pass a normal static icon or just a div? 

Even if it doesn't work in such scenarios, it should not break. Please check.